### PR TITLE
Run tests against latest IdentityModel once per day in Helix-Matrix

### PR DESCRIPTION
**Pull request by wtgodbe**
Part of https://github.com/dotnet/aspnetcore/issues/54180. Adds tagged runs of `aspnetcore-helix-matrix` against the latest available IdentityModel packages once per day.

### CODE REVIEW

File: .azure/pipelines/helix-matrix.yml [hunk 1]
```diff
@@ -31,14 +39,17 @@ jobs:
     agentOs: Windows
     timeoutInMinutes: 300
     steps:
+    - ${{ if eq(variables['Build.CronSchedule.DisplayName'], 'IdentityModel test job') }}:
+      - script: "echo ##vso[build.addbuildtag]identitymodel-test"
```
Comment (RussKie) [inline .azure/pipelines/helix-matrix.yml:43]: Shouldn't it be this?
```suggestion
      - script: echo "##vso[build.addbuildtag]identitymodel-test"
```

**Review by RussKie — APPROVED** [review]

File: .azure/pipelines/helix-matrix.yml (hunk 1 above)
Comment (wtgodbe) [reply .azure/pipelines/helix-matrix.yml:43]: This is how we do it elsewhere: https://github.com/dotnet/aspnetcore/blob/d3f5b217e5b54df6d6897b16c3ad86efd03496d9/.azure/pipelines/ci.yml#L176-L178

https://dev.azure.com/dnceng-public/public/_build?definitionId=83&tagFilter=daily-build

Comment (RussKie) [reply .azure/pipelines/helix-matrix.yml:43]: Bizarre... This is how dngecng folks been fixing the arcade:

image

Comment (wtgodbe) [reply .azure/pipelines/helix-matrix.yml:43]: Ah, looks like this approach works on Windows, but not Mac/Linux: https://github.com/dotnet/arcade/pull/14687. I'll put up a PR to fix our usage - thanks!

**Pull request merged**